net/boinc/Makefile

```diff
@@ -0,0 +1,93 @@
+# Copyright 2020 by Christian Dreihsig and Steffen Möller
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=boinc
+PKG_VERSION:=7.16.5
+PKG_VERSION_SHORT:=$(shell echo $(PKG_VERSION)| cut -f1,2 -d.)
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/BOINC/boinc
+PKG_SOURCE_DATE:=2020-02-25
+PKG_SOURCE_VERSION:=fbdcc2d6165f0a3897341852f0fcf91a934724d0
+PKG_MIRROR_HASH:=f5ad881db7fafd6fc2c5360049d4f9c95c27d41641f5a77405d7df1514ecbc9f
+
+PKG_MAINTAINER:=
+PKG_CPE_ID:=cpe:/a:boinc_project:boinc
+
+PKG_INSTALL:=1
+PKG_BUILD_PARALLEL:=0
+PKG_FIXUP:=autoreconf
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/target.mk
+
+define Package/boinc/Default
+  SECTION:=net
+  CATEGORY:=Network
+  URL:=https://github.com/BOINC/boinc/
+endef
+
+define Package/boinc/Default/description
+ The Berkeley Open Infrastructure for Network Computing (BOINC) is a
+ software platform for distributed computing: several initiatives of
+ various scientific disciplines all compete for the idle time of
+ desktop computers. The developers' web site at the University of
+ Berkeley serves as a common portal to the otherwise independently run
+ projects.
```

Comment:
two spaces, please as indentation instead of one.

Reply:
This also has its history in separate packages for libraries - merged them now.